src/pipeline.cc

```diff
@@ -465,16 +466,25 @@ class PipelineWorker : public Napi::AsyncWorker {
                   left, top, width, height, nPages, &targetPageHeight)
               : image.extract_area(left, top, width, height);
           } else {
+            int attention_x;
+            int attention_y;
+
             // Attention-based or Entropy-based crop
             MultiPageUnsupported(nPages, "Resize strategy");
             image = image.tilecache(VImage::option()
               ->set("access", VIPS_ACCESS_RANDOM)
               ->set("threaded", TRUE));
+
             image = image.smartcrop(baton->width, baton->height, VImage::option()
-              ->set("interesting", baton->position == 16 ? VIPS_INTERESTING_ENTROPY : VIPS_INTERESTING_ATTENTION));
+              ->set("interesting", baton->position == 16 ? VIPS_INTERESTING_ENTROPY : VIPS_INTERESTING_ATTENTION)
+              ->set("attention_x", &attention_x)
+              ->set("attention_y", &attention_y));
             baton->hasCropOffset = true;
             baton->cropOffsetLeft = static_cast<int>(image.xoffset());
             baton->cropOffsetTop = static_cast<int>(image.yoffset());
+            baton->hasAttentionCenter = true;
+            baton->attentionCenterX = static_cast<int>(attention_x * jpegShrinkOnLoad);
```

Comment:
I think there's also `scale` (for WebP and SVG input) to consider here. Perhaps add a test for this also?